runtime/lua/vim/filetype.lua

```diff
@@ -774,6 +774,12 @@ local extension = {
   xml = function() vim.fn["dist#ft#FTxml"]() end,
   y = function() vim.fn["dist#ft#FTy"]() end,
   zsql = function() vim.fn["dist#ft#SQL"]() end,
+  txt = function(_, bufnr)
+    --helpfiles match *.txt, but should have a modeline as last line
+    if not vim.fn.getbufline(bufnr,'$')[1]:match("vim:.*ft=help") then
```

Comment:
Use the `getline` helper in this file

```suggestion
    if not getline(bufnr, -1):match("vim:.*ft=help") then
```

Reply:
Funnily enough, that helper function does not exist in my local copy that I just checked out O_o

Reply:
It's on master: https://github.com/neovim/neovim/blob/60d3bb745d72cbf7dd3d83660e6f4d8c1a9dd770/runtime/lua/vim/filetype.lua#L22-L24

Reply:
Oh well, that was just commited :) Incoming...

Reply:
I merged it a few minutes ago ;)